apps/src/applab/applab.js

```diff
@@ -687,6 +687,11 @@ Applab.init = function(config) {
     // and sized in drawDiv().
     Applab.setLevelHtml(level.levelHtml || level.startHtml || "");
 
+    if (!!config.level.projectTemplateLevelName) {
+      studioApp.displayAlert('#codeWorkspace', 350, 'warning',
```

Comment:
We could do that, though our component starts to feel a little less pure if it's rendering itself differently based on the state of the DOM.